utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -891,6 +898,13 @@ pub(crate) fn state_machine_call_with_proof<Block: BlockT, HostFns: HostFunction
 	let proof = proving_backend
 		.extract_proof()
 		.expect("A recorder was set and thus, a storage proof can be extracted; qed");
+
+	if let Some(path) = maybe_export_proof {
+		let mut file = std::fs::File::create(path)?;
+		use std::io::Write as _;
+		file.write_all(storage_proof_to_raw_json(&proof).as_bytes())?;
```

Comment:
Maybe add a log with the file name that was written to.

Reply:
It's done :)